app/src/debug/java/org/simple/clinic/di/DebugLoginModule.kt

```diff
@@ -0,0 +1,12 @@
+package org.simple.clinic.di
+
+import io.reactivex.Single
+import org.simple.clinic.login.LoginModule
+import org.simple.clinic.login.applock.AppLockConfig
+import java.util.concurrent.TimeUnit
+
+class DebugLoginModule : LoginModule() {
```

Comment:
Same here and other Dagger modules. We can forgo this change but we should raise a maintenance task for this.

Reply:
I do not like how we do our Dagger setup as well, as it causes a bunch of problems (does not handle process death; cumbersome in tests).

I have raised a maintenance task at https://www.pivotaltracker.com/story/show/167975791

Let's pick this up sometime.